css/camp.css

```diff
@@ -2733,44 +2733,44 @@ img {
 
 /* Space between grid cells, use on grid container. */
 .gap-none > * {
-  padding-right: 0; }
+  padding: 0; }
```

Comment:
@NickCiliak do you want to add padding on all sides, or just left and right?

Reply:
@lindseysnyder I think all sides would be best. The idea is to emulate the gap property, which adds space between the cells on all sides (except the outer sides of the grid, but I'm not sure of a way to do that with this type of fallback)

Reply:
@NickCiliak sounds good - just making sure that is what's desired. I think adding padding on all sides is a good fallback. There won't be a perfect 1:1 for grid fallbacks; so all we can do it make it the best equivalent we can with what's available. 👍